configuration/pih/htmlforms/mentalHealth.xml

```diff
@@ -860,9 +861,55 @@
 
             </div>
 
+            <div class="col">
+                <p>
+                    <label>
+                        <uimessage code="pihcore.mentalhealth.judgement"/>
+                    </label>
+                    <obs conceptId="CIEL:167116" style="radio" answerConceptIds="PIH:1897,CIEL:159406,CIEL:159407" answerCodes="good,fair,poor" answerSeparator=""/>
```

Comment:
answerCodes is not using the message properties.  This code will show the answers in English (good fair poor).

Choose one to solve the problem:

- Remove answerCodes.  It will show the concept preferred names:  good,fair,poor (English) and Bon,Passable,Mauvais
- Use answerCodes but add the translations to the message properties files